Toasters/index.css

```diff
@@ -0,0 +1,33 @@
+@import url("../main.css");
+@import url("../utility.css");
+
+.toaster {
+  border-radius: 0.3rem;
+  padding: 1rem 3rem 1rem 1rem;
+  font-size: 1rem;
+  margin: 1rem 0;
+  position: relative;
+  width: 100%;
```

Comment:
Just a suggestion, you can decrease the width of the toaster. I think it will look more good.

Reply:
I have given 100% width so whenever we use a toaster in-app then we will wrap one outer div which will have its own width so the toaster will take the whole width allowed to it.

Reply:
okay. That's good.